feconf.py

```diff
@@ -324,7 +325,7 @@ def get_empty_ratings():
 # correspond to owners of the app before setting this to True. If
 # SYSTEM_EMAIL_ADDRESS is not that of an app owner, email messages from this
 # address cannot be sent. If True then emails can be sent to any user.
-CAN_SEND_EMAILS = False
+CAN_SEND_EMAILS = True
```

Comment:
Don't flip this here. (We do turn it on in prod.)

Reply:
Oops, done